resources/views/news/index.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>News</h1>
+            @auth
+                <a href="{{ route('news.create') }}">Create new news</a>
+            @endauth
+            @if ($news->count())
+                @foreach ($news as $singleNews)
+                    <div>
+                        {{ $singleNews->title }}
+                        {!! Str::markdown($singleNews->text) !!}
+                        {{ $singleNews->author }}
+
+                        <a href="{{ route('news.edit', $singleNews) }}">Edit</a>
+                        <form action="{{ route('news.destroy', $singleNews) }}" method="post">
+                            @csrf
+                            @method('DELETE')
```

Comment:
If you removed method="post" from meetups/index view you should either delete it here, if that's working, or write it back in meetups.

Reply:
I still think that keeping the method="post" is a better way, but as long as you will be consistent in all forms and if that really works, i guess it's ok @MrHDOLEK .